### DISCUSSION TITLE
Does this remote get around the legacy issue of not following UI selection?

### DISCUSSION THREAD

**Discussion by Aurasphere-au** [bjoluc/cubase-mcu-midiremote#57] [Q&A] [answered]
Hi
Im desperately trying to not abandon 36 years of Cubase but Im over the dysfunction
I am hoping to be able to get MCU hardware to follow track selection change by user eg clicking a different channel 20 channels away (just an example)
Can this script successfully update the bank and channel selection on hardware?

Thanks

**Comment by bjoluc** [comment, marked as answer]
Hi, yes that's right! Although, to be precise, auto banking is not something I had to implement explicitly – it's in the MIDI Remote API out of the box :slightly_smiling_face:

**Comment by Aurasphere-au** [comment]
Well that sounds very cool and will give it a go...I just couldnt believe that the standard Mackie Control did not update when I clicked on the ui but does that mean it will respect hidden tracks ie folders as that is a big part of my workflow ie zoom of sorts
